Code/Tools/AWSNativeSDKInit/source/AWSLogSystemInterface.cpp

```diff
@@ -44,10 +45,14 @@ namespace AWSNativeSDKInit
     Aws::Utils::Logging::LogLevel AWSLogSystemInterface::GetLogLevel() const
     {
         Aws::Utils::Logging::LogLevel newLevel = m_logLevel;
-        int awsLogLevel = bg_awsLogLevel;
-        if (awsLogLevel >= 0)
+        if (auto console = AZ::Interface<AZ::IConsole>::Get(); console != nullptr)
         {
-            newLevel = static_cast<Aws::Utils::Logging::LogLevel>(awsLogLevel);
+            int awsLogLevel = -1;
+            console->GetCvarValue("bg_awsLogLevel", awsLogLevel);
```

Comment:
Nit: console->GetCvarValue(...) returns success or failure outcome. You may want to use the return value to check whether the cvar is retrieved successfully.

Reply:
Set awsLogLevel = -1 at the beginning, and check awsLogLevel >= 0 before setting new value. I don't think I need to check return success or failure?

Reply:
You should be good.
The return value I think is only really useful for logging that the CVar could not be found, which shouldn't be the case.